pubsub/redis/redis.go

```diff
@@ -502,10 +502,12 @@ func (r *redisStreams) pollNewMessagesLoop(stream string, handler pubsub.Handler
 			Consumer: r.metadata.consumerID,
 			Streams:  []string{stream, ">"},
 			Count:    int64(r.metadata.queueDepth),
-			Block:    0,
+			Block:    r.metadata.idleCheckFrequency,
```

Comment:
I went back and forth about using readTimeout here instead. What do we think?

Reply:
Well. Frequency gives the impression that this is a recurring task. Timeout seems to be more generic.

I think readTimeOut might be a better fit here.

Reply:
👍
I'll change it.